src/openzaak/conf/includes/base.py

```diff
@@ -591,6 +591,11 @@
     os.path.join(DJANGO_PROJECT_DIR, "notifications", "tests", "files"),
 ]
 
+#
+# VNG-API-COMMON
+#
+NOTIFICATIONS_DISABLED = os.getenv("NOTIFICATIONS_DISABLED", False)
```

Comment:
Please update the documentation: https://github.com/open-zaak/open-zaak/blob/main/docs/installation/config/env_config.md

Reply:
Added in https://github.com/open-zaak/open-zaak/pull/1266/commits/b07eb61588398f5786b676963e2442b9fc3ad184

I also changed the setting to use `config` and allow values such as `yes`, `True`, `1` in the .env